docs/index.md

```diff
@@ -684,25 +684,23 @@ Given Mocha's use of `Function.prototype.call` and function expressions to defin
 Take the following example:
 
 ```js
-var assert = require('chai').assert;
+const assert = require('chai').assert;
 
 function add() {
-  return Array.prototype.slice.call(arguments).reduce(function(prev, curr) {
-    return prev + curr;
-  }, 0);
+  return Array.prototype.slice.call(arguments).reduce((prev, curr) => prev + curr, 0);
 }
 
-describe('add()', function() {
-  var tests = [
+describe('add()', () => {
+  const tests = [
     {args: [1, 2], expected: 3},
     {args: [1, 2, 3], expected: 6},
     {args: [1, 2, 3, 4], expected: 10}
   ];
 
-  tests.forEach(function(test) {
-    it('correctly adds ' + test.args.length + ' args', function() {
-      var res = add.apply(null, test.args);
-      assert.equal(res, test.expected);
+  tests.forEach(({args, expected}) => {
+    it('correctly adds ' + args.length + ' args', () => {
```

Comment:
```suggestion
    it(`correctly adds ${args.length} args`, function() {
```

Mocha recommends _not_ using an arrow function for callbacks because of the presence of the context object (`this`).  you _can_ do this if you're not using `this`, but it can lead to subtle bugs in future test maintenance.  best to keep it out of the official docs